src/js/helpers/MapGraphics.ts

```diff
@@ -32,16 +34,44 @@ export function setNewGraphic({
     });
   }
 
-  const polygonOrPointSymbol = (type: string): SimpleFillSymbol =>
-    type === 'polygon' ? getCustomSymbol() : getImagerySymbol();
+  const setSymbol = (symbolType: string): any => {
+    switch (symbolType) {
+      case 'polygon':
+        return getCustomSymbol();
+      case 'point':
+        return getPointSymbol();
+    }
+  };
+
+  const setGeometry = (symbolType: string, geometry: __esri.Geometry): any => {
+    switch (symbolType) {
+      case 'polygon':
+        return new Polygon(geometry);
+      case 'point':
+        return new Point(geometry);
+    }
+  };
 
   allFeatures.forEach((feature: FeatureResult) => {
-    const symbol = (feature.geometry as any).rings
-      ? getCustomSymbol()
-      : polygonOrPointSymbol(feature.geometry.type);
+    const isPolygon =
+      (feature.geometry as any).rings || feature.geometry.type === 'polygon'
+        ? true
+        : false;
```

Comment:
not sure that we need `? true : false` that logic already evaluates to appropriate boolean.